### PR TITLE
Fix "fatal: not in a git directory" error

### PR DESCRIPTION
**Pull request by MetRonnie**
Currently causing nightly cylc-doc builds to fail https://github.com/cylc/cylc-doc/runs/1957476513?check_suite_focus=true

### CODE REVIEW

File: configure-git/action.yml [hunk 1]
```diff
@@ -8,4 +8,4 @@ runs:
       run: |
         git config --global user.name "github-actions[bot]"
         git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
-        git config push.default current
+        git config push.default current || true  # won't work if cwd is not git repo
```
Comment (oliver-sanders) [inline configure-git/action.yml:11]: What does this last line do?

Comment (MetRonnie) [reply configure-git/action.yml:11]: Allows you to just use `git push` to push the current branch without having to specify the branch name or the remote

Comment (MetRonnie) [reply configure-git/action.yml:11]: I suppose `--global` could be used instead of `|| true`

Comment (oliver-sanders) [reply configure-git/action.yml:11]: Does that mean a branch called "current" or is it whatever HEAD is pointing at?

Comment (MetRonnie) [reply configure-git/action.yml:11]: Whatever HEAD is pointing at - https://git-scm.com/docs/git-config#Documentation/git-config.txt-pushdefault

Comment (oliver-sanders) [reply configure-git/action.yml:11]: Thanks for the info.

**Review by oliver-sanders — APPROVED** [review]

**Pull request merged**